packages/sirius-web/frontend/sirius-web-application/src/omnibox/ImportLibraryDialog.tsx

```diff
@@ -0,0 +1,191 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import DialogTitle from '@mui/material/DialogTitle';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import Button from '@mui/material/Button';
+import DialogActions from '@mui/material/DialogActions';
+import { LibrariesImportTable } from './LibrariesImportTable';
+import { useEffect, useRef, useState } from 'react';
+import Typography from '@mui/material/Typography';
+import ButtonGroup from '@mui/material/ButtonGroup';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import { useImportLibraries } from './useImportLibraries';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import Grow from '@mui/material/Grow';
+import MenuItem from '@mui/material/MenuItem';
+import MenuList from '@mui/material/MenuList';
+import Paper from '@mui/material/Paper';
+import Popper from '@mui/material/Popper';
+import {
+  ImportLibraryAction,
+  ImportLibraryDialogProps,
+  ImportLibraryDialogState,
+  ImportStudioSplitButtonProps,
+  ImportStudioSplitButtonState,
+} from './ImportLibraryDialog.types';
+import { useCurrentProject } from '../views/edit-project/useCurrentProject';
+
+export const ImportLibraryDialog = ({ open, title, onClose }: ImportLibraryDialogProps) => {
+  const [state, setState] = useState<ImportLibraryDialogState>({
+    selectedLibraries: [],
+    actions: [
+      { id: 'import', label: 'Import' },
+      { id: 'copy', label: 'Copy' },
```

Comment:
As we discussed, for now this is hardcoded, but could be requested from the backend in the future